pkgs/os-specific/linux/busybox/sandbox-shell.nix

```diff
@@ -1,9 +1,15 @@
-{ busybox}:
+{ lib, stdenv, busybox, musl }:
 
 # Minimal shell for use as basic /bin/sh in sandbox builds
 busybox.override {
   enableStatic = true;
   enableMinimal = true;
+
+  useMusl =
+    # GNU hosts use glibc which is difficult to statically link against.
+    # Prefer musl if the host platform supports it.
+    stdenv.hostPlatform.isGnu && lib.meta.availableOn stdenv.hostPlatform musl;
+
```

Comment:
This is a regression of sorts I think, we should just delete `useMusl` in `busybox`